pkg/api/job_runs.go

```diff
@@ -44,14 +45,19 @@ func (runs apiRunResults) limit(req *http.Request) apiRunResults {
 type apiRunResults []apitype.JobRun
 
 // JobsRunsReportFromDB renders a filtered summary of matching jobs.
-func JobsRunsReportFromDB(dbc *db.DB, filterOpts *filter.FilterOptions, release string, pagination *apitype.Pagination) (*apitype.PaginationResult, error) {
+func JobsRunsReportFromDB(dbc *db.DB, filterOpts *filter.FilterOptions, release string, pagination *apitype.Pagination, reportEnd time.Time) (*apitype.PaginationResult, error) {
 	jobsResult := make([]apitype.JobRun, 0)
 	table := "prow_job_runs_report_matview"
 	q, err := filter.FilterableDBResult(dbc.DB.Table(table), filterOpts, apitype.JobRun{})
 	if err != nil {
 		return nil, err
 	}
-	q = q.Where("release = ?", release)
+
+	if len(release) > 0 {
+		q = q.Where("release = ?", release)
+	}
```

Comment:
Would you mind moving this to a separate PR? I would rather land just this right away.